build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -90,8 +136,31 @@ export async function npmCheckUpdates(
             throw new TypeError(`Expected an object: ${typeof result}`);
         }
 
-        for (const [pkgJsonPath, upgradedDeps] of Object.entries(result)) {
-            const jsonPath = path.join(repoPath, pkgJsonPath);
+        // npm-check-updates returns different data depending on how many packages were updated. This code detects the
+        // two main cases: a single package or multiple packages.
+        if (glob.endsWith("**")) {
+            // if (numResults > 1) {
+            for (const [pkgJsonPath, upgradedDeps] of Object.entries(result)) {
+                const jsonPath = path.join(repoPath, pkgJsonPath);
+                // eslint-disable-next-line no-await-in-loop
+                const { name } = await readJsonAsync(jsonPath);
+                const pkg = context.fullPackageMap.get(name);
+                if (pkg === undefined) {
+                    log?.warning(`Package not found in context: ${name}`);
+                    continue;
+                }
+
+                for (const [dep, newRange] of Object.entries(upgradedDeps)) {
+                    upgradeLogLines.add(`    ${dep}: '${newRange}'`);
```

Comment:
Nit: Wasn't there an indentation helper you added in another PR? Would it make sense to use that here?

Reply:
Yes! I completely missed this. Fixed.